src/cluster.c

```diff
@@ -2160,6 +2161,7 @@ int clusterProcessPacket(clusterLink *link) {
         pauseClients(now+(CLUSTER_MF_TIMEOUT*CLUSTER_MF_PAUSE_MULT));
         serverLog(LL_WARNING,"Manual failover requested by replica %.40s.",
             sender->name);
+        clusterSendPing(link, CLUSTERMSG_TYPE_PING);
```

Comment:
@trevor211 please add a comment explaining why this ping is needed.

Reply:
Wrote a comment and rebased.